_field-types/mapping-parameters/search-analyzer.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: default
+title: Search analyzer
+parent: Mapping parameters
+grand_parent: Mapping and field types
+nav_order: 160
+has_children: false
+has_toc: false
+---
+
+# Search analyzer
+
+The `search_analyzer` mapping parameter specifies the analyzer to be used at search time for a text field. This allows the analyzer used for indexing to differ from the one used for searching, offering greater control over how search terms are interpreted and matched.
+
+By default, the same analyzer is used for both indexing and searching. However, using a custom `search_analyzer` can be helpful when you want to apply looser or stricter matching rules during search, such as using [`stemming`]({{site.url}}{{site.baseurl}}/analyzers/stemming/) or removing stopwords only at search time.
```

Comment:
We should highlight the performance impact for this during search as it will additional computation to search queries

Reply:
Thats added now